libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/DefaultFilterBar.tsx

```diff
@@ -45,6 +45,10 @@ export const DefaultFilterBarInner = (): JSX.Element => {
         customFilterName,
     };
 
+    const filterFactories = useMemo(() => {
```

Comment:
Is this necessary in the end? I thought we discovered that this was not necessry yesterday?